Src/coffeescript/cql-execution/src/example/age.coffee

```diff
@@ -26,70 +26,80 @@ module.exports = {
                "highClosed" : false,
                "type" : "Interval",
                "low" : {
-                  "name" : "DateTime",
-                  "type" : "FunctionRef",
-                  "operand" : [ {
+                  "type" : "DateTime",
```

Comment:
I noticed that you changed the ELM (age.coffee) but not the CQL (age.cql).  Was the ELM just out of date (not recently translated)?